### ISSUE TITLE
A little question about wavlm and seed mask

### ISSUE THREAD

**Issue by RobinWitch** [YoungSeng/DiffuseStyleGesture#23]
Hello, thanks for sharing code !
In the DiffuseStyleGesture , the model only use one audio feature , wavlm . 
But when extract wavlm feature from raw wav , the [code](https://github.com/YoungSeng/DiffuseStyleGesture/blob/master/main/mydiffusion_zeggs/data_loader/data_preprocessor.py#L113C9-L113C9) did't norm wav beform extract wavlm feature , while DiffuseStyleGesture++ do [it](https://github.com/YoungSeng/DiffuseStyleGesture/blob/DiffuseStyleGesturePlus/BEAT-TWH-main/process/process_BEAT_bvh.py#L161) .. . , I think DiffuseStyleGesture++ is right ,while DiffuseStyleGesture only use wavlm feature extracted from wav not norm but still generate good result in sample , it confuses me a lot . 

In the DiffuseStyleGesture ++ . the paper "_The DiffuseStyleGesture+ entry to the GENEA Challenge 2023_"  Figure 1 mentioned that seed gesture should use a mask while the [code](https://github.com/YoungSeng/DiffuseStyleGesture/blob/5362ef1650e2e54662b3081a3fe7bcedecbfd445/BEAT-TWH-main/model/mdm.py#L188C18-L188C18) not .

**Comment by YoungSeng** [comment]
Yeah, you're right. The WavLM for the first question should preferably be normalized; for the second question this RM is not quite the same as in the paper, but it should have little impact on the results, as stylization as well as controllability are not considered in the competition. I have updated in [Readme](https://github.com/YoungSeng/DiffuseStyleGesture/blob/master/BEAT-TWH-main/Readme.md#5-issues). Thanks for the correction.

**Comment by RobinWitch** [comment]
Maybe I don't explain my first question well , I suppose that using not normalized wav to get WavLM is a fatal mistake ,why it works in the end, it's so strange.... While I didn't test use normalizd wav to get WavLM in DiffuseStyleGesture...

**Comment by YoungSeng** [comment]
I'm guessing that's just a feature extractor, and the bias is the same if none of them are normalized

**Comment by RobinWitch** [comment]
Hum...still sounds strange :(
May be we need to make a simple test.

**Comment by jiaqiAA** [comment]
Hi, I think it norm wav beform extract wavlm feature, the code is `cmd = ['ffmpeg-normalize', wav_file, '-o', normalize_wav_path, '-ar', '16000']` in [here](https://github.com/YoungSeng/DiffuseStyleGesture/blob/DiffuseStyleGesturePlus/main/mydiffusion_zeggs/zeggs_data_to_lmdb.py#L133). But it is different from the [code](https://github.com/YoungSeng/DiffuseStyleGesture/blob/DiffuseStyleGesturePlus/BEAT-TWH-main/process/process_BEAT_bvh.py#L161), I don't know if it works.

**Comment by RobinWitch** [comment]
Well ,I find that mean and std of the wav is not 0 and 1 .The command`ffmpeg-normalize` is not same 
role as layer_norm ,you can refer to [it](https://github.com/slhck/ffmpeg-normalize) .

> Hi, I think it norm wav beform extract wavlm feature, the code is `cmd = ['ffmpeg-normalize', wav_file, '-o', normalize_wav_path, '-ar', '16000']` in [here](https://github.com/YoungSeng/DiffuseStyleGesture/blob/DiffuseStyleGesturePlus/main/mydiffusion_zeggs/zeggs_data_to_lmdb.py#L133). But it is different from the [code](https://github.com/YoungSeng/DiffuseStyleGesture/blob/DiffuseStyleGesturePlus/BEAT-TWH-main/process/process_BEAT_bvh.py#L161), I don't know if it works.
